tech/languages/java/java-build-tools-installation.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Java Build Tools
+page: java
+---
+
+## Maven installation
+
+Maven is probably the most popular build tool in Java world these days. It's available in official Fedora repositories, in form of RPM. To install it, simply type:
+
+```
+# dnf install maven
+```
```

Comment:
> I guess that the Fedora developer portal shouldn't be about teaching users how to use Maven, right?

I am the biggest advocate of this, but perhaps the most common install command is good to mention (for instance for Python/pip and Ruby/RubyGems it makes sense especially when comparing to installation of RPMs... I know that since we won't have that for Java, it seems less important).